### PR TITLE
Cloudera isn't using ecommerce role, so don't run corresponding role …

### PR DESCRIPTION
**Pull request by itsjeyd**
…(unless SANDBOX_ENABLE_ECOMMERCE is true).

cf. [OC-4621](https://tasks.opencraft.com/browse/OC-4621)

**Reviewers**

- [x] @pomegranited

### CODE REVIEW

File: playbooks/edx_sandbox.yml [hunk 1]
```diff
@@ -53,6 +53,7 @@
       celery_worker: True
     - edxapp
     - ecommerce
+      when: SANDBOX_ENABLE_ECOMMERCE
```
Comment (pomegranited) [inline playbooks/edx_sandbox.yml]: Also need to disable the nginx site above.

**Comment by itsjeyd** [comment]
@pomegranited Amended from9ea211a2a2c57344f52f96bfb68ad09ec69df025.

File: playbooks/edx_sandbox.yml [hunk 2]
```diff
@@ -53,6 +52,7 @@
       celery_worker: True
     - edxapp
     - ecommerce
```
Comment (pomegranited) [inline playbooks/edx_sandbox.yml]: Needs to add the `role:` label to allow the `when:` clause to be added:
```yaml
- role: ecommerce
  when: SANDBOX_ENABLE_ECOMMERCE
```

**Comment by pomegranited** [comment]
@itsjeyd  :+1: once my changes are addressed.  Thanks 😄 

- [ ] I didn't test this, because it will be tested with the appserver redeployment.
- [x] I read through the code especially carefully given that I didn't test it.  Ran it through a yaml lint checker, and compared it with the current [master version](https://github.com/edx/configuration/blob/master/playbooks/edx_sandbox.yml).
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A

**Comment by itsjeyd** [comment]
Amended from 59694f28a5c65edb35b9b75091677492f7e8a432.

File: playbooks/edx_sandbox.yml (hunk 2 above)
Comment (itsjeyd) [reply playbooks/edx_sandbox.yml]: @pomegranited Ack, sorry about the back-and-forth here. I've been trying to squeeze this in while focusing on something else, which *I know* is not the best idea, even for small changes like this... 😑 Anyway, d65cc437641b985099c27ca2399aa6bd47d71e45 should do what we need.

**Comment by itsjeyd** [comment]
Thanks @pomegranited! Merging now.

**Pull request merged**